core/src/main/java/tech/pegasys/web3signer/core/service/jsonrpc/handlers/PassThroughHandler.java

```diff
@@ -52,6 +62,17 @@ public void handle(final RoutingContext context) {
         request.method(), headersToSend, request.path(), context.getBodyAsString());
   }
 
+  private boolean isRpc(final RoutingContext context) {
+    final HttpMethod method = context.request().method();
+    try {
+      final JsonRpcRequest request =
+          jsonDecoder.decodeValue(context.body().buffer(), JsonRpcRequest.class);
+      return method.equals(HttpMethod.POST) && request.getVersion().equals("2.0");
```

Comment:
nit: Is being able to deserialize to JsonRpcRequest sufficient to return true? 

Taking that even further, perhaps could use `mapper.canDeserialize()` though not sure that saves anything compared to full deserialization

Reply:
For Ethereum it always versions "2.0" with a post request, so that is why I added the additional constraint.

Looking at `mapper.canDeserialize()` https://javadoc.io/static/com.fasterxml.jackson.core/jackson-databind/2.15.1/com/fasterxml/jackson/databind/ObjectMapper.html#canDeserialize-com.fasterxml.jackson.databind.JavaType-. It doesn't look it will do what I want. It will tell me whether Jackson knows whether understands how to deserialise a JsonRPC, that is it has the appropriate deserialiser. But it won't tell me if the body request can be deserialised as a jsonRPC. It doesn't take any input just a type.